packages/react/src/primitives/shared/styleUtils.ts

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
```

Comment:
any reason to use `*` instead of default export?

Reply:
Yes, this is the new recommended way to import React. See:
https://epicreact.dev/importing-react-through-the-ages/

TLDR (my understanding): React doesn't actually expose a default export. We've been able to do this previously by using the [esModuleInterop](https://www.typescriptlang.org/tsconfig#esModuleInterop) setting which enables [allowSyntheticDefaultImports](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports), which converts `import React from 'react'` to `import * as React from 'react'`. Doing a namespace import is ESM compatible and future proof.



https://twitter.com/housecor/status/1386839524362407943?lang=en

Reply:
discussed offline: we'll change default exports in a separate PR